changelog/v1.18.0-rc3/kubegateway-clusternames.yaml

```diff
@@ -0,0 +1,10 @@
+changelog:
+  - type: BREAKING_CHANGE
+    issueLink: https://github.com/solo-io/solo-projects/issues/7105
+    resolvesIssue: false
+    description: >-
+      When using the Kubernetes Gateway API, envoy cluster names corresponding to Kubernetes Service Upstreams
+      are now in a new more parseable format that allows us to extract info about the Service for metrics:
+      `upstreamName_upstreamNs_svcNs_svcName_svcPort` (underscore-separated). Note that as long as Kubernetes
```

Comment:
Is this up-to-date: `upstreamName_upstreamNs_svcNs_svcName_svcPort`?

Reply:
yes!